docs/getting-started.md

```diff
@@ -346,445 +252,365 @@ const ctype = Kilt.CType.fromSchema({
   },
   type: 'object',
 })
+console.log('CType: ')
+console.log(JSON.stringify(ctype, undefined, 2))
 ```
 
-### 3.2. Storing a CTYPE
+### 2.3 Store the CType on the KILT blockchain
 
-Before you can store the CTYPE on the blockchain, you have to configure your blockchain address and connect to it.
-
-To store the CTYPE on the blockchain with the attester, you have to call:
+With the built CType object, the attester can now create and sign a transaction that writes it on the KILT blockchain.
 
 ```typescript
-const tx = await ctype.getStoreTx()
+/* The attester signs the ctype creation transaction resulting from calling `ctype.store()` with its DID. */
```

Comment:
We should indicate that if you are not using a local chain, it is best practice to check if the CType is stored on chain